packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -489,12 +501,133 @@ export class Users {
       account: params.account,
     });
 
+    const reportingFeeDivisor = await augur.contracts.universe.getOrCacheReportingFeeDivisor_();
```

Comment:
hopefully this can be pulled from logs

Reply:
Yeah I wasn't planning to keep it long-term.